src/Sylius/Bundle/CoreBundle/Migrations/Version20220912091947.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
```

Comment:
We should use `Sylius\Bundle\CoreBundle\Doctrine\Migrations\AbstractMigration` instead to restrict the migration to MySQL only.